src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitControllerLegacy.kt

```diff
@@ -78,8 +79,8 @@ class VisitControllerLegacy(
     @PathVariable
     reference: String,
     @RequestBody @Valid
-    cancelOutcome: OutcomeDto,
+    cancelVisitDto: CancelVisitDto,
   ): VisitDto {
-    return visitService.cancelVisit(reference.trim(), cancelOutcome)
+    return visitService.cancelVisit(reference.trim(), cancelVisitDto)
```

Comment:
do we want to put it on legacy, as syscon may be still using it, I think they might have started using the new rest end point probably worth checking! I've asked @lbennett-moj  about it!

Reply:
thanks.. Yes I will speak to syscon